ui/src/dashboards/components/DashboardsPageContents.tsx

```diff
@@ -27,12 +22,12 @@ interface Props {
   onExportDashboard: (dashboard: Dashboard) => () => void
   onImportDashboard: (dashboard: Dashboard) => void
   notify: (message: Notification) => void
-  showOverlay: ShowOverlayActionCreator
   dashboardLink: string
 }
 
 interface State {
   searchTerm: string
+  showOverlay: boolean
```

Comment:
how do you feel about changing this variable to be prefixed with is to identify it as a boolean when reading it? such as `isOverlayShown`?

Reply:
I never know what to name these things. I like your suggestion much better